src/page/dataverse/component/generalMetadata/generalMetadata.tsx

```diff
@@ -0,0 +1,57 @@
+import type { FC } from 'react'
+import { memo } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { IconName } from '@/component/icon/icon'
+import { Icon } from '@/component/icon/icon'
+import './generalMetadata.scss'
+import './i18n/index'
+
+export type GeneralMetadataTitle =
+  | 'format'
+  | 'topic'
+  | 'license'
+  | 'geographicalCoverage'
+  | 'temporalCoverage'
+  | 'category'
+
+export type GeneralMetadata = {
```

Comment:
There are three distinct GeneralMetadata models: one for Dataspace, one for Service, and one for Dataspace. While it may seem possible to merge them into one model, doing so would result in inaccuracies in the meaning and nature of the conveyed information. Therefore, it is advisable to rely on the concepts of the ontology for consistency as much as possible.

Reply:
Definitively the second option. But, please take a step back, and consider the element to be displayed as a metadata property (and not a general metadata title), which describes a property (what you call the title), its value (what you call description), to which you associate an icon, eventually:

```ts
type MetadataProperty = {
  property: string
  value: string
  iconName?: IconName
}
```

You may not know this yet, but ontology properties, like the format property (`core:hasFormat`) are also described as such, and therefore have a label property (`rdfs:label`), internationalized, which specifies the label to use when representing the property. This is why you can't use: `t(`generalMetadata.${title}`)`. So it makes sense that the property is a string, since the value conveys the exact representation of the property, for the current local language.

Now, another issue arises with the values. You are assuming that it is a string, but that is not necessarily the case. For instance, the temporal coverage is a date range, and the dates must be represented according to the locale. And this is the responsibility of the component, imho... This is just fantastic 😅